src/sas/qtgui/Perspectives/Invariant/InvariantPerspective.py

```diff
@@ -365,24 +455,53 @@ def calculate_thread(self, extrapolation: str) -> None:
 
         # High Q Extrapolation calculations
         if self._high_extrapolate:
-            function_high = "power_law"
+            function_high: str = "power_law"
             if self._high_fit:
                 self._high_power_value = None
+            elif self._high_fix:
+                self._high_power_value = float(self.model.item(WIDGETS.W_HIGHQ_POWER_VALUE_EX).text())
+            else:
+                logger.debug("High Q extrapolation: no fit/fix option selected, using default power value")
+                self._high_power_value = DEFAULT_POWER_VALUE
```

Comment:
Is it possible to hit this clause - surely one of fit/fix must always be selected?